build/configs/artik053/st_things/defconfig

```diff
@@ -793,14 +793,21 @@ CONFIG_NET_SECURITY_TLS=y
 CONFIG_TLS_MPI_MAX_SIZE=512
 
 #
-# HW Selection
-#
+# SE Selection
+#
+CONFIG_SE=y
```

Comment:
To enable SE, seclink related configs are also needed. That causes an build error as noted by travis.